src/jsonprovider/jsonprovider.cpp

```diff
@@ -0,0 +1,289 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <utils/exception.hpp>
+#include <utils/json.hpp>
+
+#include <Poco/JSON/Parser.h>
+
+#include "jsonprovider/jsonprovider.hpp"
+
+/***********************************************************************************************************************
+ * Statics
+ **********************************************************************************************************************/
+
+namespace {
+
+static aos::DeviceInfo DeviceInfoFromJSON(const aos::common::utils::CaseInsensitiveObjectWrapper& object)
```

Comment:
Is static necessary for unnamed namespace?

Reply:
Nope